src/pages/DetailsPage.js

```diff
@@ -108,6 +108,10 @@ class DetailsPage extends React.PureComponent {
             pronouns = this.props.translate(`pronouns.${localeKey}`);
         }
 
+        const phoneNumber = getPhoneNumber(details);
+        const displayName = isSMSLogin ? this.props.formatPhoneNumber(phoneNumber) : details.displayName;
```

Comment:
Won't this make it so that their displayName will show up as their phone number even if they have a name set. For example, if I have my primary login on my account as `1234567890` and my display name as `Puneet Lath` this will make it so that my display name shows as `(123) 456-7890`. I think we only want it to show as that if I don't have a display name set.

Reply:
It is the current [behavior](https://github.com/Expensify/App/blob/d196a1217754ffab2e10ba28ba3c88f0b6391bf5/src/pages/DetailsPage.js#L93), I just moved the [condition](https://github.com/Expensify/App/blob/d196a1217754ffab2e10ba28ba3c88f0b6391bf5/src/pages/DetailsPage.js#L130) up and changed the formatting number function for a new one. I can ofc add a check if there is a name and display it instead of the phone number, but should it be done in this issue?

Reply:
Ah ok got it. Let's handle separately then. 

@koko57 would you want to raise a separate PR for that fix?

Reply:
@rushatgabhane I can take it :) So if you create an issue, please assign me

Reply:
Do you need an account you can sign in with or just one you can message? If you just need to message one, you can message my +19726585619 account. I can take care of any tests that require logging in.

Reply:
Just responded!